docs/user/tutorials/01-70-rate-limit.md

```diff
@@ -0,0 +1,400 @@
+# Tutorials - Rate Limit
+
+RateLimit CR allows creating local rate limit configuration for specific paths for the exposed application.
+
+## Prerequisites
+
+This tutorial works on kubernetes cluster with istio-operator and api-gateway-operator installed.
+Follow the installation of each module before moving forward.
```

Comment:
```suggestion
* You have the Istio and API Gateway modules added.
* You have [set up your custom domain](./01-10-setup-custom-domain-for-workload.md). Alternatively, you can use the default domain of your Kyma cluster and the default Gateway `kyma-system/kyma-gateway`.
  
  > [!NOTE]
  > Because the default Kyma domain is a wildcard domain, which uses a simple TLS Gateway, it is recommended that you set up your custom domain for use in a production environment.

  > [!TIP]
  > To learn what the default domain of your Kyma cluster is, run `kubectl get gateway -n kyma-system kyma-gateway -o jsonpath='{.spec.servers[0].hosts}`.

```